File: Source/ses_io/src/wrappers/fortran/wrap_ses_get_material_id.f90


#undef DEBUG_WRAP

function ses_get_material_id_f90(the_material_name)

   implicit none

   SES_MATERIAL_ID :: ses_get_material_id_f90
   SES_STRING :: the_material_name

   SES_MATERIAL_ID :: ses_get_material_id_c

#ifdef DEBUG_WRAP
   print *, "ses_get_material_id_f90:  the_name is ", TRIM(the_material_name)
#endif
   ses_get_material_id_f90 = ses_get_material_id_c(TRIM(the_material_name), ses_get_material_id_f90)
#ifdef DEBUG_WRAP
    print *, "ses_get_material_id_f90:  return_value is ", ses_get_material_id_f90
#endif 
end function ses_get_material_id_f90

File: Source/ses_io/src/wrappers/fortran/wrap_ses_get_material_id.c


#include "ses_defines.h"
#include "ses_globals.h"
#include "ses_internals.h"
#include "ses_externs.h"

#include "stdio.h"
#include <stdlib.h>

#include <stdio.h>

#undef DEBUG_WRAP

#ifdef LC_UNDER
ses_material_id ses_get_material_id_c_(ses_string the_material_name) {
#else

#ifdef UC_UNDER
ses_material_id SES_GET_MATERIAL_ID_C_(ses_string the_material_name) {
#else

#ifdef LC_NOUNDER
ses_material_id ses_get_material_id_c(ses_string the_material_name) {
#else

#ifdef UC_NOUNDER
ses_material_id SES_GET_MATERIAL_ID_c(ses_string the_material_name) {
#endif

#endif
#endif
#endif

#ifdef DEBUG_WRAP
  printf("wrap_ses_get_material_id.c:  the_material_name  is %s\n", the_material_name);
#endif

  ses_material_id return_value = 0;
/*  removed from user interface

  ses_material_id return_value = ses_get_material_id(the_material_name);

*/

#ifdef DEBUG_WRAP
  printf("wrap_ses_get_material_id.c:  return_value is %d\n", return_value);
#endif

  return return_value;
 
}

 
